samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java

```diff
@@ -17,19 +32,27 @@
 
 import org.json.JSONObject;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Response;
+
 public class MainActivity extends AppCompatActivity {
 
     private ChatClient chatClient;
     private int eventHandlerCalled;
     private JSONObject eventPayload;
 
     // Replace <userToken> with your valid communication service token
-    private final String userAccessToken = "<userToken>";
-    private final String endpoint = "https://chat-sdktester-e2e.int.communication.azure.net";
+    private final String userAccessToken = "eyJhbGciOiJSUzI1NiIsImtpZCI6IjEwMl9pbnQiLCJ4NXQiOiJnMTROVjRoSzJKUklPYk15YUUyOUxFU1FKRk0iLCJ0eXAiOiJKV1QifQ.eyJza3lwZWlkIjoiYWNzOjQ2ODQ5NTM0LWViMDgtNGFiNy1iZGU3LWMzNjkyOGNkMTU0N18wMDAwMDAwOC03YjVhLThiNzgtMTY1NS0zNzNhMGQwMDliYTEiLCJzY3AiOjE3OTIsImNzaSI6IjE2MTQyOTQ5MzYiLCJpYXQiOjE2MTQyOTQ5MzYsImV4cCI6MTYxNDM4MTMzNiwiYWNzU2NvcGUiOiJjaGF0IiwicmVzb3VyY2VJZCI6IjQ2ODQ5NTM0LWViMDgtNGFiNy1iZGU3LWMzNjkyOGNkMTU0NyJ9.e03hLzcAoXWYW_lyMmK8ixf5s4K7k3gIccOs7Ykm5JUvA57CgrZx8SD19uM4NI2dq8O2Yhuf_p1RX7UyT1VJmuS6_zqLKtJfq2eqNTErFcb9d2UjjL8rJPr_TJMX3CZBxi65FgSrIUu1dHGFjkR4EVsViGo64zbKl3XDfMdFlgessg6jAfHCsBsD6XUMhl54w3GYmLtKydpyAaHzaqRBjSpLaZZeFucPAvs86COX_bqhYzg93bLel5iDKWmAI1bl0sFFqcDdPYwpiypWn2xKq5XXwHc9a7gpwiAldEAzQDC7O6e32AtoDfYWppaB5bTDg4PyiP35y_yiM3WH-y_QvQ";
```

Comment:
Is this a valid token? We do not want to commit credentials to our repositories at any point in time.

Reply:
Token expires in 24 hours, but this one I accidentally commit, let me revert it.